packages/desktop-client/src/browser-preload.browser.js

```diff
@@ -50,28 +50,26 @@ global.Actual = {
     window.location.reload();
   },
 
-  openFileDialog: async ({ filters = [], properties }) => {
+  openFileDialog: async (filter = []) => {
```

Comment:
This API is intended to mirror the Electron `openFileDialog` API, which will be called directly when running in the Electron app: https://github.com/actualbudget/actual/blob/92ddaa568ac9a337755ba46598a4c20952d30048/packages/desktop-electron/index.js#L314-319

Reply:
Whoops, I see! I'll revert that change then.

Reply:
Removed 👍